xikolo-ios/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -106,6 +122,15 @@ class CourseListViewController : AbstractCourseListViewController {
         }
     }
 
+    @IBAction func unwindLogin(_ segue: UIStoryboardSegue) {
```

Comment:
I'm not sure about the unwind segue. Alternatively we could here also listen to the `loginStateChanged` notification and change the content if necessary.